packages/core/src/schema/models/commits.ts

```diff
@@ -26,6 +28,12 @@ export const commits = latitudeSchema.table(
     ),
     title: varchar('title', { length: 256 }).notNull(),
     description: text('description'),
+    authorId: text('author_id')
+      .notNull()
+      .references(() => users.id, { onDelete: 'cascade' }),
+    workspaceId: bigserial('workspace_id', { mode: 'bigint' })
+      .notNull()
+      .references(() => workspaces.id, { onDelete: 'cascade' }),
```

Comment:
Instead of having to specify that this column is a `bigserial`, can't I just infer the type from `workspaces.id` since it's already defined?

Reply:
I think you can

Reply:
Hm there's no official way of doing this. Weird, I think it makes hella sense